### PR TITLE
docs: (Core) add links to the responsive breadcrumb section

### PR DESCRIPTION
**Pull request by Lokanathan-k**
#### Please provide a link to the associated issue.
Closes: https://github.com/SAP/fundamental-ngx/issues/3114

#### Please provide a brief summary of this pull request.

adding links to the responsive breadcrumb section

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [x] update `README.md`
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Review by droshev — CHANGES_REQUESTED** [review]
I don't think this should be `fix:`

File: apps/docs/src/app/core/component-docs/breadcrumb/examples/breadcrumb-responsive-example.component.html [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <div #responsiveBreadcrumbContainer>
     <fd-breadcrumb [containerElement]="responsiveBreadcrumbContainer">
         <fd-breadcrumb-item>
-            <a fd-breadcrumb-link>Breadcrumb Level 1</a>
+            <a fd-breadcrumb-link [attr.href]="'#'">Breadcrumb Level 1</a>
```
Comment (InnaAtanasova) [inline apps/docs/src/app/core/component-docs/breadcrumb/examples/breadcrumb-responsive-example.component.html:4]: Shouldn't it be `attr.href="#"`?

Comment (Betrozov) [reply apps/docs/src/app/core/component-docs/breadcrumb/examples/breadcrumb-responsive-example.component.html:4]: or just `href="#"`

Comment (Lokanathan-k) [reply apps/docs/src/app/core/component-docs/breadcrumb/examples/breadcrumb-responsive-example.component.html]: fixed

**Review by olshansky — APPROVED** [review]

**Review by Betrozov — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Comment by puru-hk** [comment]
Tested it is working as expected.

**Review by puru-hk — COMMENTED** [review]
Looks good

**Pull request merged**